pkg/lib/cockpit-components-dropdown.jsx

```diff
@@ -43,6 +43,7 @@ export const KebabDropdown = ({ dropdownItems, position, isDisabled, props }) =>
             onSelect={() => setKebabOpen(!isKebabOpen)}
             toggle={(toggleRef) => (
                 <MenuToggle
+                    id={toggleButtonId || null}
```

Comment:
Why the `null`? Isn't `undefined` a bit more canonical? (Not a blocker, though)

Reply:
Right, but then we can drop it. So re-pushed :)